modules/microshift-oc-mirror-list-ops-catalogs.adoc

```diff
@@ -0,0 +1,56 @@
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-oc-mirror-list-operators-catalogs_{context}"]
+= Using oc mirror to select catalogs and list Operators
+
+Use the following example procedure to select a catalog and list Operators from available {OCP} content to add to your oc mirror image set configuration file.
+
+[NOTE]
+====
+To use your own catalogs and Operators, you can push the images directly to your internal registry.
+====
+
+.Prerequisites
+* The {oc-first} is installed.
+* Operator Lifecycle Manager (OLM) is installed.
+* The oc-mirror OpenShift CLI (oc) plugin is installed.
+
+.Procedure
+. Get a list of available {OCP} catalogs to filter by running the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc mirror list operators --version {product-version} --catalogs
+----
+
+. Get a list of Operators in the {OCP} catalog by running the following example command:
++
+[source,terminal]
+----
+$ oc mirror list operators <--catalog=registry.redhat.io/redhat/redhat-operator-index:v{product-version}> <1>
+----
+<1> You can replace _<--catalog=registry.redhat.io/redhat/redhat-operator-index:v{product-version}>_ with the path to your own Catalog as required. For example, `--catalog=quay.io/operatorhubio/catalog:latest`.
+
+. Select an Operator. For this example, `amq-broker-rhel9` is selected.
+
+. Optional. To select the exact channel and version of the Operator you want, list the characteristics by using the following example commands:
+
+.. Get a list of channels by running the following command:
++
+[source,terminal]
+----
+$ oc mirror list operators --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.15 --package=amq-broker-rhel9
```

Comment:
```suggestion
$ oc mirror list operators --catalog=registry.redhat.io/redhat/redhat-operator-index:v4.15 --package=amq-broker-rhel8
```

What about `{product-version}` instead of `4.15`?

Reply:
I can use an attribute, but then will the operator not match? Would a rhel8 operator version be compatible with OCP 4.17, for example?

Reply:
I don't know, but there's only `amq-broker-rhel9` and OCP is using RHEL9 for couple version already. We can keep it at hardcoded